src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -16,6 +16,106 @@
 		<var name="ns-bibo" value="http://purl.org/ontology/bibo/"/>
 	</vars>
 	<macros>
+		<macro name="setRdfSubjectDescribedBy">
+			<combine name="~rdf:subject" value="${a}">
+				<data source="$[field]"/>
+				<data source="@id" name="a">
+					<regexp match="(.*)" format="$[ns-lobid-resource]${1}"/>
+				</data>
+			</combine>
+		</macro>
+		<macro name="setProvenance">
+			<combine name="$[field_1]" value="$[ns-lobid-organisation]${a}#!" flushWith="$[field_2]">
+				<choose name="a" flushWith="$[field_2]">
+					<data source="$[field_2]" name="a">
+						<lookup in="sigel2isilMap"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<!-- Off = Offline -->
+						<regexp match="[hH][bB][zZ]/[oO][fF][fF]" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<!-- OZK = Online-Zentralkatalog" -->
+						<regexp match="HBZ/OZK" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="[hH][bB][zZ]/[rR][eE][dD]" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBibAlt" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBibOff" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBib" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^NRW/Hist.Buch$" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^NRW$" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^[dD][nN][bB]$" format="DE-101"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^BSZ$" format="DE-576"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^BVB$" format="DE-604"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^GBV$" format="DE-601"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^HBZ$" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^HeBIS$" format="DE-603"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="^KOBV$" format="DE-602"/>
```

Comment:
I was reluctant to add the abbrevations because they are no real sigels. But since there also sigels of ebook packages and national licenses, I add them now.